_includes/_head.html

```diff
@@ -1,36 +1,88 @@
-<head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+{%- assign base_url = site.domain | append: site.baseurl -%}
+{%- assign full_page_url = base_url | append: page.url -%}
+
```

Comment:
These two variables are used for other vars below. This is only for readability.